src/stdx.zig

```diff
@@ -520,3 +520,46 @@ pub fn fstatfs(fd: i32, statfs_buf: *StatFs) usize {
         @intFromPtr(statfs_buf),
     );
 }
+
+// TODO(Zig): https://github.com/ziglang/zig/issues/17592.
+/// True if every value of the type `T` has a unique bit pattern representing it.
+/// In other words, `T` has no unused bits and no padding.
+pub fn hasUniqueRepresentation(comptime T: type) bool {
```

Comment:
When vendoring, we've been adopting our snake_case convention.
Arguable here, since we should move back to std as soon we update Zig again 🤔.

```suggestion
pub fn has_unique_representation(comptime T: type) bool {
```

Reply:
>  Arguable here, since we should move back to std as soon we update Zig again 🤔.

It's a small enough change either way, so I'm happy to adopt it.